ethrpc/interface.go

```diff
@@ -61,13 +61,18 @@ import (
 // eth_getWork
 // eth_submitWork
 // eth_submitHashrate
-//
-//
-// Unstandard JSON-RPC Methods:
-//
-// eth_getBlockRange -- Optimism
 
+// RawInterface also returns the bytes of the response body payload
+type RawInterface interface {
```

Comment:
Wouldn't this be an extension of `Interface` and thus `RawInterface` should include the base? 🤔

Reply:
will do :)